alacritty/src/display.rs

```diff
@@ -329,6 +340,18 @@ impl Display {
         });
     }
 
+    /// Enable subpixel anti-aliasing
+    fn enable_font_smoothing(enable: bool) {
+        unsafe {
+            let key = NSString::alloc(nil).init_str("CGFontRenderingFontSmoothingDisabled");
+            if enable {
+                id::standardUserDefaults().setBool_forKey_(NO, key);
+            } else {
+                id::standardUserDefaults().removeObject_forKey_(key);
+            }
+        }
+    }
```

Comment:
I'd like to move this part to the font crate, since this is where the font smoothing is configured and used.

Reply:
Into the `font/src/darwin/mod.rs` file, eg?

Reply:
Yeah. There are a few approaches we could take, either we check whenever a font is loaded or we provide a method as part of its public API.